routes/delete_condition/delete_condition.controller.js

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+const { getSessionData } = require('./../../utils')
+
+module.exports = (app, route) => {
+  route.draw(app).post((req, res) => {
+    const data = getSessionData(req)
+    const index = req.params.id - 1 // array is zero-indexed
+
+    data.conditions.splice(index, 1)
```

Comment:
Might want to consider splitting out into a `deleteItemAtIndex` method or using `data.conditions = lodash.pullAt(index)` ([documentation for lodash.pullAt](https://lodash.com/docs/4.17.15#pullAt))since we are using lodash methods for other things in the app, and this logic will be used for deleting medications and treatments as well.